server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/GetVacationResponseMethod.java

```diff
@@ -64,27 +64,27 @@ public GetVacationResponseMethod(VacationRepository vacationRepository, ZonedDat
     }
 
     @Override
-    public Stream<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession) {
+    public Flux<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession) {
         Preconditions.checkNotNull(request);
         Preconditions.checkNotNull(methodCallId);
         Preconditions.checkNotNull(mailboxSession);
         Preconditions.checkArgument(request instanceof GetVacationRequest);
 
+        return Flux.from(metricFactory.runPublishingTimerMetricLogP99(JMAP_PREFIX + METHOD_NAME.getName(),
+            process(mailboxSession)
+                .flatMapMany(response -> Flux.just(JmapResponse.builder()
```

Comment:
why Flux and not Mono?

Reply:
because in draft a method can generate several responses

Reply:
however you can `flatMap` then call `flux()`